### PR TITLE
Implement issue #10

### PR DESCRIPTION
**Pull request by GregDThomas**
Provide support for configuration settings to disable propagation of the mousewheel event and also the speed of scrolling. Also include a couple of examples showing these.

### CODE REVIEW

File: src/perfect-scrollbar.js [hunk 1]
```diff
@@ -2,7 +2,25 @@
  * Licensed under the MIT License
  */
 ((function($) {
-    $.fn.perfectScrollbar = function(option) {
+
+	// The default settings for the plugin
```
Comment (hatashiro) [inline src/perfect-scrollbar.js:6]: tab sizes!

File: examples/options-wheelSpeed.htm [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+		<title>perfect-scrollbar example - use wheelSpeed to change speed of scrolling</title>
+        <link href="../src/perfect-scrollbar.css" rel="stylesheet">
```
Comment (hatashiro) [inline examples/options-wheelSpeed.htm:6]: indent!

File: examples/options-wheelSpeed.htm [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.0/jquery.min.js"></script>
+        <script src="../src/jquery.mousewheel.js"></script>
+        <script src="../src/perfect-scrollbar.js"></script>
+        <style>
+            .contentHolder { position:relative; margin:0px auto; padding:0px; width: 600px; height: 400px; overflow: hidden; }
+            .contentHolder .content { background-image: url('./azusa.jpg'); width: 1280px; height: 720px; }
+			.spacer { text-align:center }
```
Comment (hatashiro) [inline examples/options-wheelSpeed.htm:13]: indent!

**Comment by hatashiro** [comment]
Hi!

First of all, I'm really appreciated of your contributions!
It really helps me.
I'm kind of busy at my workplace nowadays and have not enough time to look into this.
As I see the patch, the contents are nearly perfect.

But I'm really sorry but here're some requests.
They're not about the contents, but about code convention issues.
1. Please divide the patch into two separated patches, like:
   1) Add optional settings feature.
   2) Add code examples.
2. Change the extensions of the example html files.
   I'd like to use .html instead of .htm.
3. Please check indentations.
   I'm using "tabsize 4" and "expand tab to spaces" for JavaScript sources in this project.
   For example html files, "tabsize 2" is okay, but it seems that some indents are wrong in the head section.

Thank you very much again.

Regards.

**Comment by GregDThomas** [comment]
Hi, no problem, will look at this as soon as I can, hopefully before the end of the week.

**Comment by hatashiro** [comment]
I pick the patch and changed them by myself.
Thank you!

**Comment by GregDThomas** [comment]
Apologies; work dragged me off to Boston at short notice (from the UK!) so never quite got around to looking at this again.

On a related note, have you thought about using Grunt as part of the packaging process; this should pick up the style-type issues that tripped me up here. I'm happy to make the changes myself if you're interested.

**Pull request closed without merging**